src/test/java/com/spatineo/anonymisator/FullStackTest.java

```diff
@@ -132,6 +132,7 @@ public void testRowWithNumberLooksLikeIPButNotIP() throws Exception
 		}
 		System.out.println(result);
 
+		//The anonymiser cannot tell whether the string 8.15.1.0 is an IP address or not and thus the expected result here is that the version number is anonymised
```

Comment:
Please use javadoc format:

`
/**
 * The anonymiser cannot tell whether the string 8.15.1.0 is an IP address or not and thus the expected result here is that the version number is anonymised
 **/`